app/assets/src/components/views/samples/samples_view.scss

```diff
@@ -13,6 +13,8 @@
     flex: 1 0 auto;
     display: flex;
     flex-direction: column;
+    width: 100%;
+    height: 100%;
```

Comment:
You should not need this... and can get tricky to mix flex grow with setting height this way. Why is height affecting this issue?

Reply:
The height is affecting this issue because it gets stuck once you expand the page to the entire window, just like the width does. If I just add in `width: 100%`, the bug persists

Reply:
I can't figure out why it works. I'm trying to find out why

Reply:
Found why adding the `height:100%` worked. At first the AutoSizer was setting a height of 0. After reading `usingAutoSizer.md` from reactVirtualized, I found that:

> AutoSizer expands to fill its parent but it will not stretch the parent. This is done to prevent problems with flexbox layouts. If AutoSizer is reporting a height (or width) of 0- then it's likely that the parent element (or one of its parents) has a height of 0. One easy way to test this is to add a style property (eg background-color: red;) to the parent to ensure that it is the correct size. (eg You may need to add height: 100% or flex: 1 to the parent.)

Reply:
i guess my question is why the parent was not stretching. It already has flex-grow of 1 - the first parameter of `flex: 1 0 auto`.

Reply:
I have no clue why it wasn't stretching. What's really weird is that `.table, .map` is under `.container` which contains `display: flex;
  flex-direction: column;
  height: 100%;
  flex: 1 0 auto;`

Reply:
Yeah, seems like a weird interaction with autosizer. Let me take a quick look later today, just to make sure there are no secondary effects.

Reply:
Sounds good! Thanks for your comments!

Reply:
This works a bit by chance... 
This only works because we are setting a minimal height for the `dataContainer` element outside. In the samples view, the toolbar takes over most of that space and the fact that the header of the table does not fit in the remainder of the space seem to affect how the table is rendered.
For instance if you inspect the banner you will see that the element overlaps with the header of the table.

Let's instead try changing the `div.tableContainer` in `BaseTable` to set `flex-basis` based on the 
header height like:

```
<div className={cs.tableContainer} style={{flexBasis: defaultHeaderHeight}}>
```

I think this will enforce that minimum size required for the table.
Let me know if it works